src/main/java/edu/ithaca/dragon/par/io/JsonDatastore.java

```diff
@@ -34,24 +37,69 @@ public JsonDatastore(String questionFilePath, String studentModelFilePath) throw
 
     @Override
     public List<StudentModel> loadStudentModels() throws IOException {
+         //check if filepath exists
         if(studentModelFilePath == null)
             throw new IOException("studentModelFilePath is null");
-        List<StudentModelRecord> studentModelRecords = JsonUtil.listFromJsonFile(studentModelFilePath, StudentModelRecord.class);
+
+        //make a list of studentModelRecords from every file in the filepath
+        List<StudentModelRecord> studentModelRecords = new ArrayList<>();
+        File studentModelFolder = new File(studentModelFilePath);
+        File[] listOfStudentModelFiles = studentModelFolder.listFiles();
```

Comment:
I think there is someway of checking that this is a directory, we should test what happens if it is given a bad path path to a file (should throw IOException and note the problem).